fec.config.js

```diff
@@ -0,0 +1,27 @@
+const { resolve } = require('path');
+
+module.exports = {
+    appUrl: '/insights/patch',
+    debug: true,
+    useProxy: process.env.PROXY === 'true',
+    proxyVerbose: true,
+    plugins: [],
+    ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
+    ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+    moduleFederation: {
+        shared: [
+            {
+                'react-router-dom': {
+                    singleton: true,
+                    import: false,
+                    version: '^6.8.1',
```

Comment:
The `version` will default to the `package.json` version, so this can be removed.

Reply:
Oh, right, Ok. Then let's leave it for now, but I'll investigate why we have this guard in the frontend components federated modules module.

Reply:
Will merge once, there is an approval